SPEC.md

```diff
@@ -436,187 +436,37 @@ The Data Connect service responds with the following table:
-    *   Trigonometric Functions:
-        *   `acos(x)` → `double`
-        *   `asin(x)` → `double`
-        *   `atan(x)` → `double`
-        *   `atan2(y, x)` → `double`
-        *   `cos(x)` → `double`
-        *   `sin(x)` → `double`
-        *   `tan(x)` → `double`
-*   **Bitwise Functions**
-    *   `bitwise_and(x, y)` → `bigint`
-    *   `bitwise_or(x, y)` → `bigint`
-    *   `bitwise_xor(x, y)` → `bigint`
-    *   `bitwise_not(x)` → `bigint`
-    *   `bitwise_left_shift(value, shift)` → [same as value]
-    *   `bitwise_right_shift(value, shift, digits)` → [same as value]
-    *   `bit_count(x, bits)` → `bigint`*
-*   **Regular Expression Functions**
-    *   `regexp_extract_all(string, pattern)` -> `array(varchar)`*
-    *   `regexp_extract_all(string, pattern, group)` -> `array(varchar)`*
-    *   `regexp_extract(string, pattern)` → `varchar`*
-    *   `regexp_extract(string, pattern, group)` → `varchar`*
-    *   `regexp_like(string, pattern)` → `boolean`*
-    *   `regexp_replace(string, pattern)` → `varchar`*
-    *   `regexp_replace(string, pattern, replacement)` → `varchar`*
-    *   `regexp_replace(string, pattern, function)` → `varchar`*
-*   **UUID Functions**
-    *   `uuid()*`
-*   **Session Information Functions**
-    *   `current_user`*
+    *   `IF`
 *   **String manipulation**
-    *   **Operators:**
-        *   `Concatenation (||)`*
-        *   `LIKE`
-    *   **Functions:**
-        *   `chr(n)` → `varchar`*
-        *   `codepoint(string)` → `integer`*
-        *   `format(format, args...)` → `varchar`
-        *   `length(string)` → `bigint`
-        *   `lower(string)` → `varchar`
-        *   `lpad(string, size, padstring)` → `varchar`
-        *   `ltrim(string)` → `varchar`
-        *   `position(substring IN string)` → `bigint`*
-        *   `replace(string, search, replace)` → `varchar`
-        *   `reverse(string)` → `varchar`
-        *   `rpad(string, size, padstring)` → `varchar`
-        *   `rtrim(string)` → `varchar`
-        *   `split(string, delimiter, limit)` -> `array(varchar)`*
-        *   `starts_with(string, substring)` → `boolean`*
-        *   `strpos(string, substring)` → `bigint`*
-        *   `substr(string, start)` → `varchar`*
-        *   `substring(string, start)` → `varchar`
-        *   `substr(string, start, length)` → `varchar`*
-        *   `substring(string, start, length)` → `varchar`
-        *   `trim(string)` → `varchar`
-        *   `upper(string)` → `varchar`
+    *   `substring(string, start)` → `varchar`
 *   **Date manipulation** 
-**Be aware of different quotation (‘) syntax requirements between MySQL and PostgreSQL. BigQuery does not support the +/- operators for dates. Convenience methods could be replaced with EXTRACT().**
-    *   **Operators:**
-        *   `+`, `- *`
-        *   `AT TIME ZONE`*
-    *   **Functions:**
-        *   `current_date`
-        *   `current_time`
-        *   `current_timestamp`
-        *   `current_timestamp(p)`*
```

Comment:
We'll need these (or now()) along with date arithmetic to convert a date of birth to an age at time of observation.

Reply:
Added